homeassistant/components/luftdaten/sensor.py

```diff
@@ -1,98 +1,125 @@
 """Support for Luftdaten sensors."""
-from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
-from homeassistant.const import ATTR_LATITUDE, ATTR_LONGITUDE, CONF_SHOW_ON_MAP
-from homeassistant.core import callback
-from homeassistant.helpers.dispatcher import async_dispatcher_connect
-
-from . import DATA_LUFTDATEN, DATA_LUFTDATEN_CLIENT, DOMAIN, SENSOR_TYPES, TOPIC_UPDATE
-from .const import ATTR_SENSOR_ID
-
-
-async def async_setup_entry(hass, entry, async_add_entities):
+from homeassistant.components.sensor import (
```

Comment:
Is this missing the annotations import?
`from __future__ import annotations`

Reply:
missed that the CI failed there... oops! Thanks, @epenet